src/types.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * PWA assets generation and injection options.
+ */
+export interface PWAAssets {
+  /**
```

Comment:
As I say, the user should not know about sharp, pwa-assets and all of these low-level libraries that are being used under the hood, everything should be enabled and done out of the box, so they just inject the plugin and everything would work.